bootstrap/kubeadm/types/v1beta2/conversion.go

```diff
@@ -84,3 +84,9 @@ func Convert_v1beta2_JoinControlPlane_To_v1alpha4_JoinControlPlane(in *JoinContr
 	// JoinControlPlane.CertificateKey exists in v1beta2 types but not in bootstrapv1.JoinControlPlane (Cluster API does not uses automatic copy certs). Ignoring when converting.
 	return autoConvert_v1beta2_JoinControlPlane_To_v1alpha4_JoinControlPlane(in, out, s)
 }
+
+func Convert_v1beta2_DNS_To_v1alpha4_DNS(in *DNS, out *bootstrapv1.DNS, s apimachineryconversion.Scope) error {
+	out.ImageRepository = in.ImageRepository
+	out.ImageTag = in.ImageTag
```

Comment:
Similar comment for v1beta1 conversions (this should not be necessary, please add a note about the dropped info)